src/main/java/Alvis/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package Alvis;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private LocalDateTime deadline;
```

Comment:
Would it better to rename this variable, maybe something like "due"? So that others don't misunderstand this as an instance of the Deadline class

Reply:
That is a good point! I had taken your suggestion and renamed it to `by`.